README.md

```diff
@@ -74,7 +76,12 @@ New major version indicates that there is an incompatible breaking change.
 With Helm v3, CRDs created by this chart can not be updated, cf the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions). Please read carefully release notes of this chart before upgrading CRDs.
 
 ```bash
-kubectl apply --server-side --force-conflicts -k https://github.com/traefik/traefik-helm-chart/traefik/crds/
+kubectl apply --server-side --force-conflicts -f https://github.com/traefik/traefik-helm-chart/traefik/crds/
+```
+
+for traefik v3, please use
+```bash
+kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/traefik/traefik/v3.0/docs/content/reference/dynamic-configuration/kubernetes-crd-definition-v1.yml
```

Comment:
I am not sure it's a good idea.  
It means that CRDs installed with the chart can be different when user upgrade. 
I understand you are trying to provide a way to install only Traefik v3 CRDs, which is a good idea, but it should provide consistent behavior between fresh install and upgrade.

Reply:
Not sure what to do about that since they decided to cut the CRDs in half, and even add an extra one for v3.
Leaving the containo.us as a valid CRD in the system without traefik monitoring it anymore seems like a good way to have people extremely confused about why nothing works anymore, and no errors if they copy some resource from some other place and dont immediately get an error

Reply:
I changed the wording some, and refered people to the migration guide

Reply:
Considering how we can handle CRDs with Helm... it seems like we should fork and provide a `traefik3-helm-chart`... :thinking:

Reply:
I think it is more an issue that belongs upstream rather than for the helm. I do think that a banner on https://github.com/traefik/traefik-helm-chart/issues about the issue should prevent too many bug reports on the helm.